src/main/webapp/i18n/de/error.json

```diff
@@ -90,6 +90,7 @@
         "systemNotificationNeedsBothDates": "Sowohl das Mitteilungsdatum als auch das Ablaufdatum müssen gesetzt sein.",
         "systemNotificationNeedsNotificationBeforeExpiration": "Das Ablaufdatum muss nach dem Mitteilungsdatum liegen.",
         "ltiNotSupported": "LTI wird auf dieser Artemis-Instanz nicht unterstützt",
-        "exportFailed": "Aufgabenexport fehlgeschlagen"
+        "exportFailed": "Aufgabenexport fehlgeschlagen",
+        "unableToImportProgrammingExercise": "Fehler beim Importieren der Programmieraufgabe."
```

Comment:
The translation is passed as a response from the server, but no translation in the client was preset